tests/sql_parse_tests.py

```diff
@@ -187,7 +189,8 @@ def test_select_if(self):
     # SHOW TABLES ((FROM | IN) qualifiedName)? (LIKE pattern=STRING)?
     def test_show_tables(self):
         query = "SHOW TABLES FROM s1 like '%order%'"
-        self.assertEqual(set(), self.extract_tables(query))
+        # TODO: figure out what should code do here
```

Comment:
When I bumped from `0.3.0` to `0.3.1`, the comments and the test case weren't super clear to me. It would be more transparent to state that this test should return an empty set, but on version `0.3.0` it incorrectly returns the database/schema name. Also it would be good to include the expected true test case here commented out, so that the next person knows that the test is expected to fail now, but commenting out the true case should work when bumping to `>=0.3.2`

Reply:
done